### PR TITLE
Added missing text for messages and removed required tags

### PR DESCRIPTION
**Pull request by rhiGriff**

### CODE REVIEW

File: frontstage/templates/message-error-summary.html [hunk 1]
```diff
@@ -10,6 +10,14 @@ <h1 class="panel__title venus">There are {{ errors|length }} errors on this page
                 {% endif %}
             </div>
             <div class="panel__body" data-qa="error-body">
+                {% if errors|length == 1 %}
+
+                <p>This <b>must be corrected </b>to continue.</p>
+
```
Comment (rjweeks) [inline frontstage/templates/message-error-summary.html]: Remove these blank lines

File: frontstage/templates/secure-messages-draft.html [hunk 2]
```diff
@@ -46,8 +46,7 @@ <h1 class="saturn">Edit a draft message</h1>
                                 class="secure-message-form__subject input input--text"
                                 type="text"
                                 name="secure-message-subject"
-                                value="{{draft['subject']}}"
-                                required/>
+                                value="{{draft['subject']}}"/>
                         </fieldset>
```
Comment (rjweeks) [inline frontstage/templates/secure-messages-draft.html]: The pattern we follow is {{ draft['subject'] }} so spaces after {{ and before }}

File: frontstage/templates/message-error-summary.html [hunk 3]
```diff
@@ -10,6 +10,12 @@ <h1 class="panel__title venus">There are {{ errors|length }} errors on this page
                 {% endif %}
             </div>
             <div class="panel__body" data-qa="error-body">
+                {% if errors|length == 1 %}
+                <p>This <b>must be corrected </b>to continue.</p>
```
Comment (dcdarrell9) [inline frontstage/templates/message-error-summary.html]: Should we indent with the "if's". It's in a lot of different places, not just here.

File: frontstage/templates/secure-messages-draft.html [hunk 4]
```diff
@@ -46,8 +46,7 @@ <h1 class="saturn">Edit a draft message</h1>
                                 class="secure-message-form__subject input input--text"
                                 type="text"
                                 name="secure-message-subject"
-                                value="{{draft['subject']}}"
-                                required/>
+                                value="{{draft['subject']}}"/>
```
Comment (andrewmil) [inline frontstage/templates/secure-messages-draft.html]: Should this be `{{ draft['subject'] }}`?

Should probably at least keep it consistent with the other changes.

File: frontstage/templates/message-error-summary.html (hunk 3 above)
Comment (andrewmil) [inline frontstage/templates/message-error-summary.html]: I think the standard is not to, it's more important to keep the html elements in line

**Pull request merged**